PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+                                        
+                                        <item xml:id="fol2" resp="AY">
+                                            <locus from="1r" to="62v"></locus>
+                                            Foliation in pencil in the right of the bottom corner
+                                            of the recto. This  foliation is used as reference in this record.
+                                        </item>
+                                    </list>
+                                </foliation>
+                                
+                                <collation>
+                                      <list>
+                                        <item xml:id="q1" n="1">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="1r" to="8v"/>
+                                        </item>
+                                        <item xml:id="q2" n="2">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="9r" to="16v"/>
+                                        </item>
+                                        <item xml:id="q3" n="3">
+                                        <dim unit="leaf">8</dim>
+                                            <locus from="17r" to="24v"/>
+                                        </item>
+                                        <item xml:id="q4" n="4">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="25r" to="32v"/>
+                                        </item>
+                                        <item xml:id="q5" n="5">
+                                           <dim unit="leaf">8</dim>
+                                            <locus from="33r" to="40v"/>
+                                        </item>
+                                        <item xml:id="q6" n="6">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="41r" to="48v"/>
+                                        </item>
+                                        <item xml:id="q7" n="7">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="49r" to="56v"/>
+                                        </item>
+                                        <item xml:id="q8" n="8">
+                                         <dim unit="leaf">6</dim>
+                                            <locus from="57r" to="62v"/>
+                                        </item>
+                                       </list>
+                                </collation>
+                                <condition key="good">The beginning and ending of the <ref target="#ms_i1"></ref>s missing and the binding is missing. 
```

Comment:
The beginning and **end** of `<ref target="#ms_i1"/>` and the binding **are** missing.